{
    "type": "bundle",
    "id": "bundle--ffdce5b5-ca81-48d8-b57e-1b0a7fdabaed",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--ce72ef14-84aa-472d-9e66-1fc195f50560",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2024-02-22T00:15:58.785328Z",
            "modified": "2024-02-22T00:15:58.785328Z",
            "name": "CVE-2023-42823",
            "description": "The issue was resolved by sanitizing logging This issue is fixed in watchOS 10.1, macOS Sonoma 14.1, tvOS 17.1, macOS Monterey 12.7.1, iOS 16.7.2 and iPadOS 16.7.2, iOS 17.1 and iPadOS 17.1, macOS Ventura 13.6.1. An app may be able to access user-sensitive data.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2023-42823"
                }
            ]
        }
    ]
}